README.md

```diff
@@ -1,4 +1,4 @@
-# AccessLocator
+# AccessLocator test git
```

Comment:
Completely remove these commits as they are not related to your issue:
![image](https://user-images.githubusercontent.com/11838191/42720511-1a08b156-86f6-11e8-9ec2-f42353dcd9ce.png)

I would do this with the following command sequence:
1. ```git rebase -i master```
1. Remove the lines that look like "pick 123some hexidecimal6789 readme file change
1. Save and exit.
1. ```git push origin jagruti -f```
```